src/app/code/community/EasyTranslate/Connector/Model/Content/Generator/AbstractEavGenerator.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+abstract class EasyTranslate_Connector_Model_Content_Generator_AbstractEavGenerator
+    extends EasyTranslate_Connector_Model_Content_Generator_AbstractGenerator
+{
+    protected $_sortedAttributeCodes = [];
+
+    protected function _getAttributeCodes(Mage_Core_Model_Abstract $model): array
+    {
+        $attributeSetId = (int)$model->getData('attribute_set_id');
+        $resourceModel  = $model->getResource();
+        if (!$attributeSetId || !$resourceModel instanceof Mage_Eav_Model_Entity_Abstract) {
+            return $this->_attributeCodes;
```

Comment:
As you added `_getAttributeCodes`, wouldn't it be cleaner to return this function here instead of directly the property?

Reply:
I first did not get your comment, because I thought it would lead to an infinite loop, but `parent::_getAttributeCodes($model)` indeed makes more sense :+1: